weave-js/src/components/Panel2/PanelExpression/ConfigComponent.tsx

```diff
@@ -170,7 +171,7 @@ export const ConfigComponent: React.FC<
                         <fieldset style={{borderWidth: '1px'}}>
                           <legend>Table Query</legend>
                           <Button
-                            size="tiny"
```

Comment:
Note: these two buttons are never shown because `weavePlotEnabled` is currently hardcoded to false.

Reply:
I don't think so.